napari/plugins/io.py

```diff
@@ -101,29 +101,14 @@ def read_data_with_plugins(
 
         return layer_data or None, hookimpl
 
-    errors: List[PluginCallError] = []
-    skip_impls: List[HookImplementation] = []
     layer_data = None
-    while True:
-        result = hook_caller.call_with_result_obj(
-            path=path, _skip_impls=skip_impls
-        )
-        reader = result.result  # will raise exceptions if any occurred
-        if not reader:
-            # we're all out of reader plugins
-            break
-        try:
-            layer_data = reader(path)  # try to read data
-            if layer_data:
-                hookimpl = result.implementation
-                break
-        except Exception as exc:
-            # collect the error and log it, but don't raise it.
-            err = PluginCallError(result.implementation, cause=exc)
-            err.log(logger=logger)
-            errors.append(err)
```

Comment:
why get rid of the logging?

Reply:
(meant to select just line 123)

Reply:
Because now we’re immediately raising.  The logging was necessary when it wasn’t raised